app/retail/templates/mission.html

```diff
@@ -20,429 +20,430 @@
-          {% endblocktrans %}
-          <br />
-          {% blocktrans %}
-          <span class="text-highlight-dark-blue"> CHANGE </span> AND
-          <span class="text-highlight-pink">GROW OPEN SOURCE</span>
-          {% endblocktrans %}
-        </div>
-      </div>
+  <head>
+    {% include 'shared/head.html' %}
+    {% include 'shared/cards_pic.html' %}
+    <link rel="stylesheet" href="{% static "v2/css/mission.css" %}" />
+  </head>
+
+  <body class="{{ active }}">
+    {% include 'shared/tag_manager_2.html' %}
+    <div class="container-fluid header">
+      {% include 'shared/nav.html' %}
+      {% include 'shared/minihero.html' with h1='Mission' position_h1_below_logo=1 %}
     </div>
-  </div>
-</div>
-
-<div class="container-fluid" style="background-color: #f2f6f9;" id="we_want">
-  <div class="row">
-    <div class="col pb-4">
-      <h4 class="text-left text-uppercase">
-        {% blocktrans %}
-        <span class="text-highlight-green">We</span> Want
-        {% endblocktrans %}
-      </h4>
-    </div>
-  </div>
-
-  <div class="row animatedParent we-want">
-    <div class="col-md-12 col-lg-8 col-xl-6 animated fadeInLeftShort">
-      <div class="row mb-5">
-        <div class="col-md-10 col-lg-8 d-flex align-items-center">
-          <img
-            class="icon"
-            src="{% static "v2/images/mission/want/rocket_man.svg" %}"
-          />
-          <span class="icon-text">
-          {% blocktrans %}
-            We want to build a world in which everyone has the <strong>financial
-            leverage to leave</strong> their job.
-          {% endblocktrans %}
-          </span>
-        </div>
-      </div>
-
-      <div class="row mb-5">
-        <div class="col-md-10 col-lg-8 d-flex align-items-center">
-          <img
-            class="icon"
-            src="{% static "v2/images/mission/want/find_work.svg" %}"
-          />
-          <span class="icon-text">
-          {% blocktrans %}
-            We want it to be as <strong>easy to find work in software</strong>, as it is for an
-            Uber driver to find a ride today.
-          {% endblocktrans %}
-          </span>
-        </div>
-      </div>
 
-      <div class="row mb-5">
-        <div class="col-md-10 col-lg-8 d-flex align-items-center">
-          <img
-            class="icon"
-            src="{% static "v2/images/mission/want/incentive.svg" %}"
-          />
-          <span class="icon-text">
-            {% trans "We want to build a network with <strong>aligned incentives</strong>, especially between capital &amp; labour." %}
-          </span>
+    <div class="container-fluid" id="mission_hero">
+      <div class="row p-5 p-5 animatedParent">
+        <div class="col-12 col-lg-10 offset-md-0 offset-lg-1 d-flex flex-sm-row flex-column text-center animated fadeInLeftShort">
+          <div class="text">
+            <div class="font-bigger-4 header-text">
+              {% blocktrans %}
+              GITCOIN IS A
```

Comment:
I personally feel let's use css `text-transform: uppercase` to make the text uppercase. So that if we decide to change it in the future it is just changing one line and won't need the copy to be changed everytime. @thelostone-mc @mbeacom what do you think?

Reply:
I will change this and use text-uppercase class from bootstrap